src/components/widgets/EventWidget.tsx

```diff
@@ -0,0 +1,78 @@
+import moment from 'moment';
+import React, { useCallback, useContext } from 'react';
+import { useQuery } from 'react-apollo';
+import { StyleSheet, View } from 'react-native';
+import { NavigationScreenProp } from 'react-navigation';
+
+import { colors, consts, normalize, texts } from '../../config';
+import { graphqlFetchPolicy } from '../../helpers';
+import { useRefreshTime } from '../../hooks';
+import { calendar } from '../../icons';
+import { NetworkContext } from '../../NetworkProvider';
+import { getQuery, QUERY_TYPES } from '../../queries';
+import { Icon } from '../Icon';
+import { BoldText, RegularText } from '../Text';
+import { Touchable } from '../Touchable';
+import { Wrapper } from '../Wrapper';
+
+type Props = {
+  navigation: NavigationScreenProp<never>;
+};
+
+export const EventWidget = ({ navigation }: Props) => {
+  const refreshTime = useRefreshTime('event-widget', consts.REFRESH_INTERVALS.ONCE_A_DAY);
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+
+  const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp, refreshTime });
+
+  const currentDate = moment().format('YYYY-MM-DD');
+  const queryVariables = {
+    dateRange: [currentDate, currentDate]
+  };
+
+  const { data, loading } = useQuery(getQuery(QUERY_TYPES.EVENT_RECORDS), {
+    fetchPolicy,
+    variables: queryVariables
+  });
+
+  const onPress = useCallback(() => {
+    navigation.navigate('Index', {
+      title: 'Veranstaltungen',
+      query: QUERY_TYPES.EVENT_RECORDS,
+      queryVariables,
+      rootRouteName: 'EventRecords'
+    });
+  }, [navigation]);
+
+  const eventCount = data?.eventRecords?.length;
+
+  return (
+    <Touchable onPress={onPress}>
+      <Wrapper>
+        <View style={styles.container}>
+          <Icon style={styles.icon} xml={calendar(colors.primary)} />
+          <BoldText style={styles.count}>{loading ? ' ' : eventCount}</BoldText>
+        </View>
+        <RegularText style={styles.text}>{texts.homeTitles.events}</RegularText>
```

Comment:
you can add the prop primary for setting the color instead of in stylesheet

Reply:
done in c9ab08812de7540f17593e724e8cbe47fe520152